src/app/forms/frontcountry-cabins/frontcountry-cabins.component.ts

```diff
@@ -78,16 +81,30 @@ export class FrontcountryCabinsComponent extends BaseFormComponent {
         notes: this.form.get('varianceNotesControl'),
       });
 
+    if (this.loading) {
+      this.disable();
+    } else {
+      // In case we init form after service is done fetching for some reason.
+      this.enable();
+    }
+
+    if (this.loading) {
+      this.disable();
+    } else {
+      // In case we init form after service is done fetching for some reason.
+      this.enable();
+    }
```

Comment:
duplicate of the above?
this is all going to get moved to `BaseFormComponent` soon either way.